x-pack/plugins/cases/public/components/user_action_tree/index.tsx

```diff
@@ -592,6 +602,55 @@ export const UserActionTree = React.memo(
 
     const comments = [...userActions, ...bottomActions];
 
+    useEffect(() => {
+      const setInitialLensComment = async () => {
+        const currentAppId = await currentAppId$.pipe(first()).toPromise();
+
+        if (!currentAppId) {
+          return;
+        }
+
+        const incomingEmbeddablePackage = embeddable
+          ?.getStateTransfer()
+          .getIncomingEmbeddablePackage(currentAppId);
+
+        if (incomingEmbeddablePackage) {
+          let draftComment;
+          if (storage.get(DRAFT_COMMENT_STORAGE_ID)) {
+            try {
+              draftComment = JSON.parse(storage.get(DRAFT_COMMENT_STORAGE_ID));
+              // eslint-disable-next-line no-empty
+            } catch (e) {}
+          }
+
+          if (draftComment?.commentId) {
+            if (
+              ![NEW_ID, DESCRIPTION_ID].includes(draftComment.commentId) &&
+              !manageMarkdownEditIds.includes(draftComment.commentId)
+            ) {
+              setManageMarkdownEditIds([draftComment.commentId]);
```

Comment:
we can add all this logic in the dispatcher to avoid dependency
```
setManageMarkdownEditIds((prevManageMarkdownEditIds) => {
  if (![NEW_ID, DESCRIPTION_ID].includes(draftComment.commentId) &&
    !manageMarkdownEditIds.includes(draftComment.commentId)) {
    return [draftComment.commentId]);
 }
 return prevManageMarkdownEditIds;
});
```